NOTICE.txt

```diff
@@ -2871,6 +2855,19 @@ SOFTWARE.
 
 ---
 
+## semver
+
+This product contains 'semver' by Semver.
+
+"Semantic Versioning" or "SemVer" contain a set of rules and requirements that dictate how version numbers are assigned and incremented
+
+* HOMEPAGE:
+  * https://github.com/semver/semver
+
+* LICENSE:
```

Comment:
No license

Reply:
That repository contains no code, so I'm not really sure why it's being included in our stuff as a dependency.

Reply:
Code appears to be unlicensed, but NPM lists it as MIT (the default).

Reply:
(I think my second comment in this thread was written on the wrong pull request, sorry)